test/test_expose.py

```diff
@@ -491,3 +491,42 @@ def define(cls, spec):
         self.check_ports(ExposeProcess, 'base', ['namespace'])
         self.check_ports(ExposeProcess, 'base.namespace', ['sub_one'])
         self.check_namespace_properties(BaseNamespaceProcess, 'namespace', ExposeProcess, 'base.namespace')
+
+    def test_expose_exclude_port_with_validator(self):
```

Comment:
Is this test to manifest the issue? If the change is not made, this will fail?

Reply:
Indeed, the message of the first commit gives more details on what this test is explicitly testing for: https://github.com/aiidateam/plumpy/pull/268/commits/90df601ee3a83a88e37e4e9be2683668149a9113